src/SDKs/Azure.Core/data-plane/Azure.Core.Tests/Azure.Core.Tests.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <IsPackable>false</IsPackable>
+    <ProjectGuid>{84491222-6C36-4FA7-BBAE-1FA804129151}</ProjectGuid>
+    <LangVersion>7.2</LangVersion>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="nunit" Version="3.10.1" />
+    <PackageReference Include="nunit" Version="3.10.1" />
+    <PackageReference Include="NUnit3TestAdapter" Version="3.10.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.7.2" />
+    <PackageReference Include="System.Memory" Version="4.5.1" />
+    <PackageReference Include="System.Text.Primitives" Version="0.1.0-preview1-180216-4" />
```

Comment:
This should be removed. Maybe just file an issue to do it later. In general, I will add more comments below that you might want to punt for this PR and just include in an issue. Up to you.

Reply:
Simply removing the reference to that package (`System.Text.Primitives`) doesn't work - we are using it in both the code and the tests. 

I will file an issue to track that.